### ISSUE TITLE
Give context information for translation

### ISSUE THREAD

**Issue by Chairn** [ddnet/ddnet#3868]
It would be nice to have more context information to help translater in doing better translation as sometimes you have no idea where it's supposed to appear for what reason, so you can't even test your translation :(

I have no idea of how it could be implemented though.

**Comment by heinrich5991** [comment]
If you don't know where it appears, you probably shouldn't translate it. Even for things that look "obvious". This can lead to botched translations, and probably won't be checked because "someone already translated them".

**Comment by def-** [comment]
We have context information for translations already, see for example:
```
[Demo browser]
Play
== Abspielen
```
vs
```
[Start menu]
Play
== Spielen
```

**Comment by Chairn** [comment]
I guess we should have one for every line.

**Comment by EGYT5453** [comment]
> If you don't know where it appears, you probably shouldn't translate it. Even for things that look "obvious". This can lead to botched translations, and probably won't be checked because "someone already translated them".

True. As an example: previous Ukrainian translation had "No" translated as "None", so when you had the "Are you sure?" window, you had two choices: "Yes" and "None"
